message-index/messages/GHC-87429/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Illegal datatype context
+summary: Constraints present in datatype declaration without DatatypeContexts.
+severity: error
+introduced: 9.6.1
```

Comment:
So by default, no extensions are needed to get the error. There is a possible extension to fix the error, but because it's deprecated I have not been using said error.

Reply:
I've removed the unnecessary `NoDatatypeContexts` pragma.